src/test/features/claim/routes/eligibility/valid-defendant.ts

```diff
@@ -71,19 +72,18 @@ describe('Claim eligibility: single claimant page', () => {
         await request(app)
           .post(pagePath)
           .set('Cookie', `${cookieName}=ABC`)
-          .send({ singleClaimant: YesNoOption.YES.option })
+          .send({ validDefendant: ValidDefendant.PERSONAL_CLAIM.option })
           .expect(res => expect(res).to.be.redirect.toLocation(pageRedirect))
       })
-
       it('should redirect to not eligible page when form is valid and not eligible option selected', async () => {
```

Comment:
Can you add test for third radio option?

Reply:
👍